style.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+h1 {
+  font-size: 20px;
+  color: #172b4d;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  margin: 130px 0 10px 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin-right: 100px;
+  padding-right: 100px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #172b4d;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.logo {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  color: #6070ff;
+  position: relative;
+  text-decoration: none;
+}
+
+h4 {
+  font-weight: 700;
+  line-height: 15px;
+  letter-spacing: 0;
+  color: #7f8cff;
+}
+
+ul {
+  display: inline-block;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.content {
+  background-image: url(header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  height: 100vh;
+}
```

Comment:
- [ ] The background image is not showing on your mobile website. Kindly fix the issue. :+1: